README.md

```diff
@@ -137,7 +137,7 @@ brew install sfml zlib libpng freetype glew
 
 - Windows:
 ```
-
+```
```

Comment:
If you could please add the missing setup command:
```
vcpkg install sfml zlib libpng freetype glew
```

Reply:
added!